docs/_utils/parse_db_properties.py

```diff
@@ -0,0 +1,37 @@
+import re
+import yaml
+import sys
+
+def process_config(config_file, output_file):
+    with open(config_file, 'r') as file:
+        content = file.read()
+
+    pattern = r',\s*(\w+)\(this,\s*"(\w+)",\s*(?:liveness::(\w+),\s*)?value_status::(\w+),\s*([^,]+),\s*"([^"]+)"\)'
+    matches = re.findall(pattern, content)
+
+    properties = []
+    for match in matches:
+        property_data = {
+            "name": match[1],
+            "value_status": match[3], 
+            "default": match[4],
+            "liveness": 'True' if match[2] else 'False',
+            "description": match[5]
+        }
+        properties.append(property_data)
+
+    # Write to YAML
+    with open(output_file, 'w') as file:
+        yaml.dump(properties, file)
```

Comment:
could just dump to `sys.stdout` one less argument to worry about. and simpler this way.

Reply:
Agree, I've change the code sys.stdout and used a pipe to create the yaml file.